Lib/test/libregrtest/utils.py

```diff
@@ -210,3 +210,10 @@ def clear_caches():
     else:
         for f in typing._cleanups:
             f()
+
+    try:
```

Comment:
alternate easier to read version?

```py
if fractions := sys.modules.get('fractions'):
    fractions._hash_algorithm.cache_clear()
```

Reply:
That would break the pattern of all the previous chunks of code in this file. So unless you want to fix all of them, let's stick to Zach's version.

Reply:
I agree with both; that's a nice cleanup for another time.